server/api/index.js

```diff
@@ -3,13 +3,13 @@ module.exports = router;
 
 router.use('/users', require('./users'));
 
-router.use('/cart', require('./cart'));
+// router.use('/cart', require('./cart'));
```

Comment:
Updated on main so that you guys should have all files now. Before you merge this to main, please un-comment out so we have all routes working again.